starter_code/hangman.js

```diff
@@ -1,36 +1,68 @@
 var hangman;
 
-// function Hangman() {
-
-// }
-
-// Hangman.prototype.getWord = function () {
-
-// };
-
-// Hangman.prototype.checkIfLetter = function (keyCode) {
-
-// };
-
-// Hangman.prototype.checkClickedLetters = function (key) {
-
-// };
-
-// Hangman.prototype.addCorrectLetter = function (i) {
-
-// };
+function Hangman() {
+this.words=["amnamn", "jsfgj","jfjir"],
+this.secretWord= "",
+this.letters=[],
+this.guessedLetter="",
+this.errorsLeft=10
+
+ }
+
+ Hangman.prototype.getWord = function () {
+     var i = Math.floor(Math.random()*(this.words.length + 2));
+     return  this.secretWord += this.words[i];
+    ; 
+};
 
-// Hangman.prototype.addWrongLetter = function (letter) {
+ Hangman.prototype.checkIfLetter = function (keyCode) {
+  if (keyCode > 64 && keyCode < 91){
+    return true;
+  } 
 
-// };
+   return false; 
+ 
+};
 
-// Hangman.prototype.checkGameOver = function () {
 
-// };
+ Hangman.prototype.checkClickedLetters = function (key) {
+   if (this.letters.indexOf(key)>0){
+    return false;
+   };
+ 
+return true;
+};
 
-// Hangman.prototype.checkWinner = function () {
+Hangman.prototype.addCorrectLetter = function (i) {
+   
+    this.guessedLetter = this.secretWord[i].toUpperCase()
```

Comment:
this should be "+=" otherwise you guessedLetter variable will only have one letter as a value, so you won't be able to compare it later (in the checkWinner per exemple)